Clojure/Embeddings/RegExp (for Clojure).sublime-syntax

```diff
@@ -0,0 +1,16 @@
+%YAML 1.2
+---
+name: Regular Expression (Clojure)
+scope: source.regexp.clojure
+version: 2
+hidden: true
+
+extends: Packages/Regular Expressions/RegExp.sublime-syntax
+
+###[ CONTEXTS ]################################################################
+
+contexts:
+
+  prototype:
+    - meta_prepend: true
+    - include: Clojure.sublime-syntax#merge-conflict-markers
```

Comment:
This one makes me wonder, whether it we should rather include `merge-conflict-markers` in `base-common` context of _RegExp (basic)_ directly to catch all multi-line regexp use cases.

I can at least think of Perl being affected but likely others are as well.

Python currently injects them via `with_prototype`, which is probably not ideal anyway and could be changed to avoid redundant patterns.

Reply:
With PR #4285 this syntax can be avoided.